Add tests:

KonstantsinYurkevich opened the pull request:
1. CheckBoxesTest
2. DropDownTest
3. InputsTest
4. TypoTest
Add maven dependency for JLanguageTool
Deleted HerokuAppTest example

KonstantsinYurkevich committed:
Add tests:
1. CheckBoxesTest
2. DropDownTest
3. InputsTest
4. TypoTest
Add maven dependency for JLanguageTool
Deleted HerokuAppTest example

dzmitryrak commented on src/test/java/TypoTest.java:
```diff
@@ -0,0 +1,29 @@
+        driver.get("http://the-internet.herokuapp.com/typos");
+        JLanguageTool langTool =  new JLanguageTool(new BritishEnglish());
+        String textFromPage = driver.findElement(By.xpath("(((//div[2])//div[1])//div[1])//p[2]")).getText();
+        List<RuleMatch> matches = langTool.check(textFromPage);
```
Какая-то жесть... А давай-ка без допустим библиотек проверим на соответствие статичному тексту?

KonstantsinYurkevich replied:
Ну если без библиотек, то там нужно будет ввести правильный текст и просто за асертить его с тем что достали со страницы, и я что видимо прошляпил, у меня по имени тега не тот текст был. Не само предложение выдавало, а текст что это предложение рандомное и тд

KonstantsinYurkevich replied:
Fixed

dzmitryrak commented on src/test/java/InputsTest.java:
```diff
@@ -0,0 +1,70 @@
+    public void canInputOrNotContainSymbols() {
+        driver.get("http://the-internet.herokuapp.com/inputs");
+        WebElement input = driver.findElement(By.tagName("input"));
+        String letters = "~!@#$%^&*()_+=-{}|][;:'/?*-.,";
```
а что ты заасертил то?

KonstantsinYurkevich replied:
Вот тут я не понял как асертить, по заданию нужно было проверить можно или нет вводить буквы. До этого в заданиях было понятно, либо что то должно быть, либо нет. А тут если что то асертить то тест всегда будет зеленый и смысла нету в тесте. Либо я сильно туплю либо нужна конкретика, должны быть буквы в этом поле или нет )))

KonstantsinYurkevich replied:
fixed

KonstantsinYurkevich committed:
Reformat code:
1. InputsTest
2. TypoTest
Removed maven dependency for spellchecking
Removed in TypoTest language library and added another way of spellchecking

KonstantsinYurkevich committed:
Added tag name for InputsTest
Modified 2 last tests in InputsTest
Replaced driver.findElements in TypoTest
